### PR TITLE
fix: removing warning

### LINKED ISSUE #229: Warning  regarding `GPUInfo` import
The following line raises a warning, and soon it will be an exception

https://github.com/ansys/pyansys-tools-report/blob/d25393c54db66d2e0f85088738e4a4d4596f6cd7/src/ansys/tools/report/report.py#L150

### PR DESCRIPTION
**Pull request by germa89**
As the title.

I use the github dev website to submit this, soo.... pre-commit might complain.

Close #229

### CODE REVIEW

File: src/ansys/tools/report/report.py [hunk 1]
```diff
@@ -147,8 +147,11 @@ def __init__(
         # bug that the user is trying to report.
         if gpu:
             try:
-                from pyvista.utilities.errors import GPUInfo
+                from pyvista.report import GPUInfo
+            except ImportError:
+                from pyvista.utilities.errors import GPUInfo  # deprecated in Pyvista 0.4
```
Comment (RobPasMue) [inline src/ansys/tools/report/report.py:152]: PyVista 0.4?? That seems a while back... Are we sure about that version?

**Comment by RobPasMue** [comment]
> . pre-commit might complain.

Good thing we have pre-commit.ci enabled 😄

File: src/ansys/tools/report/report.py (hunk 1 above)
Comment (RobPasMue) [reply src/ansys/tools/report/report.py:152]: Ahh... version 0.40. Found it

Comment (RobPasMue) [inline src/ansys/tools/report/report.py:152]:
```suggestion
                from pyvista.utilities.errors import GPUInfo  # deprecated in Pyvista 0.40
```

**Review by RobPasMue — APPROVED** [review]

**Comment by germa89** [comment]
Thank you a lot @RobPasMue !! :)

**Pull request merged**